charts/keda/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
 - name: keda
-  version: 1.3.0
+  version: 2.6.2
```

Comment:
are we trying to only put chart changes not images. whats the actual benefit out of it

Reply:
The old version had api incompatibilities.

Reply:
Should we completely get rid of keda in our software?

Reply:
hah, i don't know. do we use it? do customers use it? i'm honestly not sure where we are at with this.

Reply:
If we want to get rid of keda, we should do it in a different branch. I've opened an issue to track that discussion: <https://github.com/astronomer/issues/issues/4441>